src/features/Licenses/LicensesFilters/__test__/index.test.jsx

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Provider } from 'react-redux';
```

Comment:
Please check the implementation with renderWithProvider where we can pass a mockstore. The old implementation with Provider is gonna be replaced